manifests/p/Phoner/PhonerLite/3.09/Phoner.PhonerLite.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Phoner.PhonerLite
+PackageVersion: 3.09
+PackageLocale: en-US
+Publisher: Heiko Sommerfeldt
+PackageName: PhonerLite
+PackageUrl: https://lite.phoner.de
+License: Copyright © 2022 Heiko Sommerfeldt

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate, and use the `Copyright:` tag for copyrights